malevo/swagger.yml

```diff
@@ -0,0 +1,174 @@
+swagger: '2.0'
+info:
+  description: Malevo API
+  version: 0.1.0
+  title: Malevo API
+#host: localhost
+basePath: /api/malevo
+tags:
+  - name: class
```

Comment:
what do you think about renaming the class tag into confmat? The queries are now specific to one cell of a confusion matrix for an epoch.

We could then also extend the API later to give all the off-diagonal img ids for a specific ground truth or predicted label